airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -623,7 +623,7 @@ class EksPodOperator(KubernetesPodOperator):
     :type aws_conn_id: str
     """
 
-    template_fields: Iterable[str] = {
+    template_fields: Sequence[str] = {
```

Comment:
Looks like the expression is a set here. Convert to a tuple maybe?

Reply:
Why not.